### PR TITLE
Azure put get support for windows

### PR DESCRIPTION
**Pull request by rvishureddy**
Supports Azure put get for windows.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -222,6 +224,7 @@ if (WIN32)
     find_library(ZLIB_LIB zlib_a.lib PATHS deps-build/${PLATFORM}/${VSDIR}/zlib/lib/ REQUIRED)
     find_library(AWS_CORE_LIB aws-cpp-sdk-core.lib PATHS deps-build/${PLATFORM}/${VSDIR}/aws/lib/ REQUIRED)
     find_library(AWS_S3_LIB aws-cpp-sdk-s3.lib PATHS deps-build/${PLATFORM}/${VSDIR}/aws/lib/ REQUIRED)
+	find_library(AZURE_STORAGE_LITE_LIB azure-storage-lite.lib PATHS deps-build/${PLATFORM}/${VSDIR}/azure/lib/ REQUIRED)
```
Comment (smtakeda) [inline CMakeLists.txt]: align this to other find_library

File: CMakeLists.txt [hunk 2]
```diff
@@ -247,6 +250,7 @@ if (WIN32)
             deps-build/${PLATFORM}/${VSDIR}/zlib/include
             deps-build/${PLATFORM}/${VSDIR}/aws/include
             deps-build/${PLATFORM}/${VSDIR}/cmocka/include
+			deps-build/${PLATFORM}/${VSDIR}/azure/include
```
Comment (smtakeda) [inline CMakeLists.txt]: align this to other find_library

Comment (rvishureddy) [reply CMakeLists.txt]: Sure will do

**Comment by rvishureddy** [comment]
New PR is raised with changes for Mac and Windows.

**Pull request closed without merging**